packages/manager/modules/pci/src/projects/project/training/jobs/translations/Messages_de_DE.json

```diff
@@ -43,7 +43,17 @@
   "pci_projects_project_training_jobs_common_cancel": "Abbrechen",
   "pci_projects_project_training_jobs_resubmit_success": "Der Auftrag wurde neu gestartet",
   "pci_projects_project_training_jobs_resubmit_error": "Beim Neustart des Auftrags ist ein Fehler aufgetreten: {{ message }}",
+  "pci_projects_project_training_jobs_submit_select_config_resources_tips_cpu_identifier": "vCPU",
+  "pci_projects_project_training_jobs_submit_select_config_resources_tips_gpu_identifier": "GPU",
+  "pci_projects_project_training_jobs_submit_select_data_tips_1": "Die Daten werden in ein temporäres Dateisystem in der Nähe des Auftrags geladen, um die Latenzen und die Leistung zu verbessern, bevor sie als Volume erstellt werden.",
+  "pci_projects_project_training_jobs_submit_select_data_tips_2": "Es hat sich bewährt, einen Container für Ihre Eingabedaten und einen zweiten für die Ausgabedaten hinzuzufügen.",
+  "pci_projects_project_training_jobs_submit_data_prefix": "Nach Präfix filtern",
+  "pci_projects_project_training_jobs_submit_data_prefix_help": "Nur Objekte mit dem entsprechenden Präfix werden in das Dateisystem geladen.",
+  "pci_projects_project_training_jobs_submit_data_permission_help": "Im „Nur-Lese-Modus“ wird am Ende des Auftrags keine Synchronisierung gestartet. Der Auftrag wird dadurch schneller abgeschlossen.",
+  "pci_projects_project_training_jobs_submit_data_cache": "Cache",
+  "pci_projects_project_training_jobs_submit_data_cache_help": "Container im Cache können ohne zusätzliche Synchronisierung durch andere Aufträge wiederverwendet werden. Wenn sie über einen bestimmten Zeitraum nicht verwendet werden, werden die Container aus dem Cache entfernt."
+  "pci_projects_project_training_jobs_resubmit_error": "Beim Neustart des Auftrags ist ein Fehler aufgetreten: {{ message }}",
```

Comment:
Looks like the rebase introduced some invalid JSON files here.
I'm rebasing to address them.